wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -29,7 +29,7 @@
       ]
     },
     {
-      "label": "Directives",
+      "label": "Transformation steps",
```

Comment:
Label for UDD is still User defined Directives? Or that should also be changed?

Reply:
Also there should be corresponding change in docs as well. https://github.com/data-integrations/wrangler/blob/develop/wrangler-transform/docs/Wrangler-transform.md

Reply:
@CuriousVini Good point. This change is only on the UI. I don't know changing UDD falls under the scope. @bdmogal Any suggestions? 

Will fix the doc.

Reply:
yea I was referring to UI label UDD https://github.com/data-integrations/wrangler/blob/develop/wrangler-transform/widgets/Wrangler-transform.json#L44
![image](https://user-images.githubusercontent.com/14131070/52252471-6c24fb80-28b7-11e9-85c8-29e8d5c6c927.png)